src/shared/VacationTable/VacationTable.tsx

```diff
@@ -239,66 +239,80 @@ const VacationTable: FC = () => {
     );
   };
 
-  return filterType && filtered ? (
+  const renderEmptyTable = () => {
+    return (
+      <Box display={"flex"} justifyContent={"center"}>
+        <Typography
+          sx={(theme) => ({
+            border: `1px solid ${theme.palette.primary.main} `,
+            padding: "20px",
+            borderRadius: "6px",
+          })}
+          color={"primary"}
+          variant="h4"
+        >
+          Table is empty!
+        </Typography>
+      </Box>
+    );
+  };
+
+  return filterType ? (
```

Comment:
instead of just string define type literal interface for filterType. 
It will be much easier to read this prop